v0/destinations/am/transform.js

```diff
@@ -327,6 +324,18 @@ function responseBuilderSimple(
     default:
       if (message.channel === "mobile") {
         set(payload, "device_brand", message.context.device.manufacturer);
+
+        const deviceId = get(message, "context.device.id");
+        const platform = get(message, "context.device.type");
+        const tracking = get(message, "context.device.adTrackingEnabled");
+        const advertId = get(message, "context.device.advertisingId");
+
+        if (tracking && platform.toLowerCase() === "ios") {
+          set(payload, "idfa", advertId);
+          set(payload, "idfv", deviceId);
```

Comment:
Why is this gated behind adTrackingEnabled ? The IDFV is available on all iOS devices / app settings, even if the user has not consented to ad tracking.

The IDFA, on the other hand, is much more intrusive and requires consent before it is available.

Reply:
Got it. We'll update this section.